collectors/mesos/agent/agent.go

```diff
@@ -190,7 +190,7 @@ func (c *Collector) metricsMessages() (out []producers.MetricsMessage) {
 
 		fi := getFrameworkInfoByFrameworkID(cm.FrameworkID, c.agentState.Frameworks)
 		if fi == nil {
-			c.log.Warnf("Did not find FrameworkInfo for framework ID %s, skipping!", fi.ID)
+			c.log.Warnf("Did not find FrameworkInfo for framework ID %s, skipping!", cm.FrameworkID)
```

Comment:
There are a couple of instances like these throughout this project; I lean towards leaving them as is, especially since this project is no longer being used 1.12 onwards.